test/conformance/testdata/conformance.txt

```diff
@@ -41,6 +41,7 @@ test/e2e/auth/service_accounts.go: "should allow opting out of API token automou
 test/e2e/common/configmap.go: "should be consumable via environment variable"
 test/e2e/common/configmap.go: "should be consumable via the environment"
 test/e2e/common/configmap.go: "should fail to create ConfigMap with empty key"
+test/e2e/common/configmap.go: "should patch ConfigMap successfully"
```

Comment:
Drop this, can't add and promote a test within the same PR (ref https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-testing-in-kubernetes)